dist/angular-localForage.js

```diff
@@ -14,7 +14,11 @@
       return factory(angular, localforage);
     });
   } else if(typeof exports === 'object' || typeof global === 'object') {
-    module.exports = factory(angular, require('localforage')); // Node/Browserify
+    if(typeof module === 'undefined') {
+      global.module.exports = factory(angular, require('localforage')); // NW.js
+    } else {
+      modules.exports = factory(angular, require('localforage')); // Node/Browserify
```

Comment:
Also, it looks like your fix hasn't been pushed to your remote.

From your repo:

```bash
git push
```